core/src/main/res/values/strings.xml

```diff
@@ -229,6 +229,8 @@
     <string name="download_type_media">Media file</string>
     <string name="download_type_image">Image</string>
     <string name="download_request_error_dialog_message_prefix">An error occurred when trying to download the file:\u0020</string>
+    <string name="null_value_error_dialog_message_prefix">Error</string>
```

Comment:
We already have the string `error_label` that is translated to many languages. Please use this one instead.

Reply:
ok no problem